src/components/PupilComponents/BrowseFactorSelector/BrowseFactorSelector.tsx

```diff
@@ -83,14 +85,18 @@ export const BrowseFactorSelector = ({
             <OakPupilJourneyYearButton
               phase={phaseSlug}
               key={factor.factorSlug}
-              onClick={() => onClick(factor)}
+              onClick={() => {
+                onClick(factor);
+                if (onCallback) {
+                  onCallback(factor);
```

Comment:
This extra callback looks redundant to me. All of this can be in the onClick callback